pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -89,6 +89,9 @@ var exampleXML = `<domain type="kvm" xmlns:qemu="http://libvirt.org/schemas/doma
   <features>
     <acpi></acpi>
     <smm></smm>
+    <kvm>
+      <hidden state="yes"></hidden>
```

Comment:
@crobinso shouldn't the state be `on` or `off`?

Reply:
Whoops, good catch. Fixed now